content/en/real_user_monitoring/guide/monitor-capacitor-applications-using-browser-sdk.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Monitor Capacitor Applications Using the Browser SDK
+description: Learn how to monitor cross-platform applications built with Capacitor through the browser RUM SDK.
+further_reading:
+  - link: '/real_user_monitoring/'
+    tag: 'Documentation'
+    text: 'Learn about Real User Monitoring'
+---
+
+## Overview
+
+[Capacitor][1] is an open source native runtime for building Web Native applications that run natively on iOS, Android, and progressive web applications with JavaScript, HTML, and CSS.
+
+You can install and configure the Datadog Browser SDK to start monitoring applications you've built with Capacitor. The configuration provides you with visibility into the JavaScript portion of your application (excludes native application visibility).
+
+**Note**: Applications wrapped by Capacitor to run an **iOS** target uses `capacitor://` as the default scheme to serve local assets.
+
+## Installation
+
+To install the Datadog Browser SDK to support Capacitor apps:
+
+1. Set up and install [RUM Browser Monitoring][3], following the steps for CDN sync, CDN async, or npm.
+2. Set the `allowFallbackToLocalStorage` parameter to `true` in the RUM initialization configuration.
+
+   **Note**: This setting allows Datadog to collect RUM data without relying on browser cookies.
+
+   ```javascript
+   datadogRum.init({
+     applicationId: '<DATADOG_APPLICATION_ID>',
+     clientToken: '<DATADOG_CLIENT_TOKEN>',
+     site: '<DATADOG_SITE>',
+     ...
+     allowFallbackToLocalStorage: true
+   });
+   ```
+
+3. Once you've configured the SDK correctly, your data populates the [RUM Explorer][3].
+
+## Troubleshooting
+
+### I only have visibility on the JavaScript portion of my application, but not the native part
+
+This is expected behavior. The native part of a Capacitor application, whether it is through the usage of plugins or custom code, is not monitored. Usually plugins forward a response status that can be tracked from the JavaScript side of the application. However, if a plugin crashes, or the entire application crashes due to native code issues, it is not reported to Datadog.
```

Comment:
So i saw client that instrument both native and capacitor separately, they pay double however they samples native at a very low rate just to see some instances of crashes etc. Should this be added here? We may want to consult with eng about it.

Reply:
cc @nhientrandatadog @BenoitZugmeyer @MaelNamNam Can you provide guidance here?